apps/prairielearn/src/lib/editors.ts

```diff
@@ -635,9 +635,10 @@ export class AssessmentCopyEditor extends Editor {
     debug('Write infoAssessment.json with new title and uuid');
     infoJson.title = assessmentTitle;
     infoJson.uuid = this.uuid;
-    await fs.writeJson(path.join(assessmentPath, 'infoAssessment.json'), infoJson, {
-      spaces: 4,
-    });
+
+    const formattedJson = await formatJsonWithPrettier(JSON.stringify(infoJson));
+
+    await fs.writeFile(path.join(assessmentPath, 'infoAssessment.json'), formattedJson);
```

Comment:
Nit, no need for a newline, ditto elsewhere:

```suggestion
    const formattedJson = await formatJsonWithPrettier(JSON.stringify(infoJson));
    await fs.writeFile(path.join(assessmentPath, 'infoAssessment.json'), formattedJson);
```